lib/nexus/nexus_plot.cpp

```diff
@@ -122,22 +195,24 @@ nex::NexusPlot::NexusPlot( std::istream& stream ) {
 
 void nex::NexusPlot::load(std::istream& stream) {
     struct stream_guard {
-        stream_guard( std::istream& stream ) :
-            mask( stream.exceptions() ),
-            s( stream ) {}
-        ~stream_guard() { this->s.exceptions( this->mask ); }
+        stream_guard(std::istream &stream) :
+                mask(stream.exceptions()),
+                s(stream) {}
+
+        ~stream_guard() { this->s.exceptions(this->mask); }
+
         std::ios::iostate mask;
-        std::istream& s;
-    } g { stream };
-    stream.exceptions( std::ios::goodbit );
+        std::istream &s;
+    } g{stream};
+    stream.exceptions(std::ios::goodbit);
```

Comment:
just a curiosity: I looked up the documentation for `exceptions()` but it's fairly fuzzy. What does that function do??